src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -780,7 +780,8 @@ Prevents content overflow in navbar. */
     width: 30px;
 }
 
-.ui-datepicker-calendar .ui-state-default {
+.ui-datepicker-calendar,
+.ui-state-default {
```

Comment:
I'll just leave a warning here that the comma makes a semantic difference and not just syntactic.